.github/workflows/mvn-branch-ci.yaml

```diff
@@ -1,9 +1,6 @@
-name: Build Branch
+name: Build Branch (PR)
```

Comment:
How would this be triggered ?
On raising a PR, should the Developer manually trigger the this CI Action ?

Reply:
ahh.. i see 

```
pull_request:
    branches:	    branches:
      "**"	
```

Reply:
This is triggered solely from PRs.